docs/assets/javascript/graph.js

```diff
@@ -0,0 +1,91 @@
+$('.md-sidebar--secondary').each(function() {
+    var link = $(this).html();
+    $(this).contents().append('<div id="graph" style="height:300px; width: 100%;"></div>');
+  });
+  
+  var dom = document.getElementById('graph');
+  var myChart = echarts.init(dom, null, {
+    renderer: 'canvas',
+    useDirtyRect: true
+  });
+  var option;
+  
+  $.getJSON(window.location.origin + '/data.json', function (graph) {
+    myChart.hideLoading();
+    graph.nodes.forEach(function (node) {
+      node.symbolSize += 5;
```

Comment:
min size of the nodes in the graph is 5. 
the size will be increased be each edge of the node by 1.